### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by NickHackman**
- Added `dirs` crate to `comparison`
- Added links to crates in `comparison`
- Added links to documentation for functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,17 +108,18 @@ Here is an overview of them, combined with ratings on properties that guided the
...
-| `standard_paths`      | Maintained     |  ✔  |  ✖  |  ✔  | ✔  | ✔  | ✔  | ✖  |
-| `xdg`                 | Maintained     |  ✔  |  ✖  |  ✖  | ✔  | ✖  | ✔  | 🞈  |
-| `xdg-basedir`         | Unmaintained?  |  ✔  |  ✖  |  ✖  | ✔  | ✖  | ✖  | 🞈  |
-| `xdg-rs`              | Obsolete       |  ✔  |  ✖  |  ✖  | ✔  | ✖  | ✖  | 🞈  |
+| Library            | Status        | Lin | Mac | Win | Base | User | Proj | Conv |
+| ------------------ | ------------- | :-: | :-: | :-: | :--: | :--: | :--: | :--: |
+| `app_dirs`         | Unmaintained  |  ✔  |  ✔  |  ✔  |  🞈   |  ✖   |  ✔   |  ✖   |
+| `app_dirs2`        | Maintained    |  ✔  |  ✔  |  ✔  |  🞈   |  ✖   |  ✔   |  ✖   |
+| **dirs-next**      | **Developed** |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
+| dirs               | Unmaintained  |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
```
Comment (lzutao) [inline README.md]: "This crate is a fork of unmaintained dirs crate", is a not at the beginning of readme.
This seems needless to me.

File: README.md [hunk 2]
```diff
@@ -82,24 +82,24 @@ dirs_next::executable_dir();
...
-| `public_dir`     | `Some(XDG_PUBLICSHARE_DIR)`     or `None`                                                        | `Some({FOLDERID_Public})`         | `Some($HOME`/Public/`)`                     |
-| `template_dir`   | `Some(XDG_TEMPLATES_DIR)`       or `None`                                                        | `Some({FOLDERID_Templates})`      | `None`                                      |
-| `video_dir`      | `Some(XDG_VIDEOS_DIR)`          or `None`                                                        | `Some({FOLDERID_Videos})`         | `Some($HOME`/Movies/`)`                     |
+| Function name                                                                         | Value on Linux/Redox                                                                            | Value on Windows                  | Value on macOS                              |
+| ------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- | --------------------------------- | ------------------------------------------- |
```
Comment (lzutao) [inline README.md:87]: I'm not a fan of adding links to markdown table. It makes line so long to work in vim.
Could you consider adding external links outside this table?

File: README.md [hunk 3]
```diff
@@ -108,18 +109,18 @@ Here is an overview of them, combined with ratings on properties that guided the
...
-| **dirs-next**      | **Developed** |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
-| dirs               | Unmaintained  |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
-| `directories-next` | Developed     |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✔   |  ✔   |
-| `s_app_dir`        | Unmaintained? |  ✔  |  ✖  |  🞈  |  ✖   |  ✖   |  🞈   |  ✖   |
-| `standard_paths`   | Maintained    |  ✔  |  ✖  |  ✔  |  ✔   |  ✔   |  ✔   |  ✖   |
-| `xdg`              | Maintained    |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✔   |  🞈   |
-| `xdg-basedir`      | Unmaintained? |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✖   |  🞈   |
-| `xdg-rs`           | Obsolete      |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✖   |  🞈   |
+| Library                                                         | Status        | Lin | Mac | Win | Base | User | Proj | Conv |
+| --------------------------------------------------------------- | ------------- | :-: | :-: | :-: | :--: | :--: | :--: | :--: |
```
Comment (lzutao) [inline README.md:113]: There were links to other crates there. I removed those to keep the simple.
People could go to crates.io and enter these crate names.
Also long lines are hard to work in vim.

File: README.md (hunk 1 above)
Comment (NickHackman) [reply README.md]: I don't think redundancy here hurts at all. I also see it being likely that someone skips over the vast majority of the readme, thereby missing the note.

File: README.md (hunk 2 above)
Comment (NickHackman) [reply README.md:87]: Yeah, it does make the lines a little on the long side, but it's definitely nicer to use than referencing a footer/external list

File: README.md (hunk 3 above)
Comment (NickHackman) [reply README.md:113]: Sorry, didn't realize `dirs-rs` originally had links to the other crates in the comparison. I personally like the links for the ease of looking and comparing other possible information for the crates.

**Comment by lzutao** [comment]
I'm sorry for taking so long to get into this PR. There are some management issues
for this project.

As stated in #34, the maintenance status of dirs and directories is unclear. I would 
prefer to remove them from the table for now.

Also, the line length after adding links is kinda long. Could you use below syntax instead ?

```
[Link][1]
⋮
[1]: http://b.org 
```

**Comment by lzutao** [comment]
I'm gonna close this PR has been inactive for a while. 
If you want to reopen the PR, please file a new PR.
Thanks for your contributing!

**Pull request closed without merging**